src/Sulu/Bundle/AdminBundle/Resources/js/components/AutoCompletePopover/Suggestion.js

```diff
@@ -49,13 +49,13 @@ export default class Suggestion extends React.PureComponent<Props> {
             <span>
                 {splittedText.map((splitText, index) => {
                     return (
-                        <>
+                        <div key={'suggestion.'+index}>
```

Comment:
that can not be changed to div this should stay as a fragment its not allowed to be a tag here.

Reply:
Split into #8298